OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameChatLogic.cs

```diff
@@ -39,6 +39,8 @@ public class IngameChatLogic : ChromeLogic
 		bool disableTeamChat;
 		bool teamChat;
 
+		public string ChatLine = ChromeMetrics.Get<string>("ChatLine");
```

Comment:
Could you rename this to ChatLineSound?
Of course the value in metrics.yaml then still needs to be ChatLine for backwards compatibility, but the field name would otherwise be a bit ambigous to modders.